config/opal_check_libnl.m4

```diff
@@ -0,0 +1,230 @@
+dnl -*- shell-script -*-
+dnl
+dnl Copyright (c) 2015-2016 Research Organization for Information Science
+dnl                         and Technology (RIST). All rights reserved.
+dnl $COPYRIGHT$
+dnl
+dnl Additional copyrights may follow
+dnl
+dnl $HEADER$
+dnl
+
+dnl
+dnl Helper function to parse --with-libnl* options
+dnl
+dnl $1: variable name
+dnl $2: --with-<foo> value
+dnl
+AC_DEFUN([OPAL_LIBNL_PARSE_WITH],[
```

Comment:
I'm not quite sure what the purpose of this macro is -- did we use to call this for both `libnl` and `libnl3` (i.e., that's why there's a substitution on `$1`)?

I ask because I only see this macro being called for `libnl` now, which makes me wonder why this is still a macro with `$1` and `$2` substitution...?  Indeed, there's hard-coded places below where you check the value of `$opal_want_libnl`.